### DISCUSSION TITLE
Error while using fiber.fit function

### DISCUSSION THREAD

**Discussion by CharbelSakr26** [mtex-toolbox/mtex#1660] [Ask Anything]
Dear MTEX expert,

While using the fiber.fit function to get the best fit of a fiber for an ODF, I get this  error message:
```
 "Error using ODF/mean
Too many output arguments.

Error in fibre.fit (line 225)
      [~,~,lambda,eigv] = mean(ori);" 
```

Here is an example of the code: 
```matlab
 mtexdata dubna
odf3 = calcODF(pf,'resolution',5*degree,'zero_Range')
[~,ori_pref] = max(odf3)
volume(odf3, ori_pref, 20*degree) * 100
f = fibre.fit(odf3).
```
Could you please help me in how to solve it?
Thank you.

**Comment by ralfHielscher** [comment]
I just released a new version MTEX 5.8.2 which hopefully fixes this issue.

Ralf.
